learn-more.html

```diff
@@ -1,138 +0,0 @@
-            </div>
-          </div>
-          <div class="post">
-            <a href="blog-post.html" class="buttonS">July 28</a>
-            <div class="post-head text-medium">And Another Post</div>
-            <div class="post-content text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dictum non mauris et lacinia. Quisque facilisis nibh magna, condimentum efficitur neque maximus in. Aenean ultrices gravida ante, eget tincidunt justo sollicitudin in. 
-            </div>
-          </div>
-          <div class="post">
-            <a href="blog-post.html" class="buttonS">July 26</a>
-            <div class="post-head text-medium">One More Post</div>
-            <div class="post-content text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dictum non mauris et lacinia. Quisque facilisis nibh magna, condimentum efficitur neque maximus in. Aenean ultrices gravida ante, eget tincidunt justo sollicitudin in.
-            </div>
-          </div>
-        </div>
-
-        <div id="of-note">
-          <div class="note-chunk">
-            <h1 class="note-head text-medium">Something of Note</h1>
-            <div class="note-content">
-              <div id="note-box"></div>
-              <div class="note-text text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas cursus consectetur enim. Phasellus pellentesque sit amet risus vehicula rhoncus. Suspendisse eget ullamcorper tortor. Aliquam id quam porta, aliquam urna sed, ultrices augue. Integer ut pharetra ligula, a gravida orci. Duis dui massa, tincidunt pulvinar diam ac, gravida suscipit turpis. Phasellus.</div>
-            </div>
-            <a href="learn-more.html" class="buttonL">LEARN MORE</a>
-          </div>
-          <div class="note-chunk">
-            <h1 class="note-head text-medium">Something of Note</h1>
-            <div class="note-content">
-              <div id="note-box"></div>
-              <div class="note-text text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas cursus consectetur enim. Phasellus pellentesque sit amet risus vehicula rhoncus. Suspendisse eget ullamcorper tortor. Aliquam id quam porta, aliquam urna sed, ultrices augue. Integer ut pharetra ligula, a gravida orci. Duis dui massa, tincidunt pulvinar diam ac, gravida suscipit turpis. Phasellus.</div>
-            </div>
-            <a href="learn-more.html" class="buttonL">LEARN MORE</a>
-          </div>
-        </div>
-
-      </div>
-
-    </section>
-    
-<!--- - - - - - footer - - - - -->
-    <section id="footer">
-      <div id="get-in-touch" class="text-medium">Questions or comments?<strong> Get in touch</strong></div>
-      <div id="grid-for-footer">
-        <form id="form">
-          <input type="text" placeholder="First Name" style="vertical-align: top;"></input><br>
-          <input type="text" placeholder="Last Name"></input><br>

```

Comment:
There is some controversy over whether or not to use the placeholder attribute. According to MDN, it is an [archived attribute](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Attribute/placeholder) that should not be used in [new websites](https://developer.mozilla.org/en-US/docs/Archive). Here are also a couple articles talking about this topic: [Here](https://www.smashingmagazine.com/2018/06/placeholder-attribute/) and [here](https://developer.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/).